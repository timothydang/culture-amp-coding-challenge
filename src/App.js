import React from 'react';
import { Header } from 'semantic-ui-react';
import AppRouter from './Routes';

const style = {
  h1: {
    marginTop: '1em',
    marginBottom: '1em'
  }
}

function App() {
  return (
    <div>
      <Header as="h1" content='Surveys' textAlign="center" style={style.h1}/>
      <AppRouter />
    </div>
  );
}

export default App;
