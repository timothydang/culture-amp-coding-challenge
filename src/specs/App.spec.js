import React from 'react';
import App from './../App';
import { render } from '@testing-library/react';

describe('Application root', () => {
  it('should render without crashing', () => {
    const { getByText } = render(<App />);
    expect(getByText('Surveys')).toBeInTheDocument();
  });
});
