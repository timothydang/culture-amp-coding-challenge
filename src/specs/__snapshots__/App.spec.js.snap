// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application root should render without crashing 1`] = `
<div>
  <h1
    className="ui center aligned header"
    style={
      Object {
        "marginBottom": "1em",
        "marginTop": "1em",
      }
    }
  >
    Surveys
  </h1>
  <div
    className="ui active centered inline loader"
  />
</div>
`;
